tbx/project_styleguide/templates/patterns/pages/errors/404.yaml

```diff
@@ -1,6 +0,0 @@
-context:
```

Comment:
Nitpick: you might want to keep this file and mock the footerlinks tag as we have done on other templates:

```
tags:
  footerlinks:
    '':
      template_name: 'patterns/navigation/components/footer-links.html'
```

Reply:
Oh that's a good shout. Thanks, I'll add that in.

Reply:
Added in [99b6ba5](https://github.com/torchbox/torchbox.com/pull/182/commits/99b6ba50b532c90069b8f9d7a347841405a3d85a)